docs/index.md

```diff
@@ -1 +1,84 @@
-Updated API docs coming soon!
+# API Reference
+
+## Authentication
+Every request to the Nebula Data API requires a valid API key included in the HTTP header `Authorization`.
+```HTTP
+GET /v1/example/ HTTP/1.1
```

Comment:
Change this out with some other publicly accessible route so developers new to APIs don't get confused.
```suggestion
GET /v1/sections HTTP/1.1
```